src/main/java/seedu/address/model/student/UniqueStudentList.java

```diff
@@ -125,18 +126,16 @@ public boolean hasName(Name name) {
     }
 
     /**
-     * Returns true if {@code session} with same date and time exists in the session list of the
-     * student with the {@code name}
-     * guarantees that student will not be null
+     * Returns true if {@code session} with same date and time exists in any of the students in the unique student list
```

Comment:
Not sure if capitalization of session here would make a difference.

Reply:
Thanks for spotting, will make a change to this.